src/app_service/common/network_constants.nim

```diff
@@ -264,7 +264,7 @@ var NODE_CONFIG* = %* {
   },
   "Networks": NETWORKS,
   "TorrentConfig": {
-    "Enabled": true,
+    "Enabled": false,
```

Comment:
We should also change default value here:
https://github.com/status-im/status-desktop/blob/821b54fda66aa5cad1f2c7581d527756f5e50410/src/app_service/service/accounts/service.nim#L397

But that's only in `master` branch